packages/@lwc/state/src/index.ts

```diff
@@ -122,6 +122,24 @@ const update: MakeUpdate = <
   };
 };
 
+function createStateManagerAtomsClosure() {
+  const smAtoms = new WeakSet();
+
+  return {
+    atom: <T,>(v: T) => {
+      const a = atom(v);
+      smAtoms.add(a);
+
+      return a;
+    },
+    setAtom: <T = unknown>(a: AtomSignal<T>, v: T) => {
+      if (smAtoms.has(a)) {
+        a[atomSetter](v);
+      }
```

Comment:
should we throw an error here if `! smAtoms.has(a)`? silently ignoring the set call seems like it would be hard to debug